### PR TITLE
Updating doc contribution

### PR DESCRIPTION
**Pull request by mikeroySoft**
I wanted the team to review before just publishing this, as it describes how to contribute to the documentation given the new workflows we have.

Can folks take a look at DOCS-CONTRIBUTING.md and let me know if there's anything from the repo's perspective that we should change?

This PR also includes a change to make the docs directly editable from the doc site without having to clone and fork, as well as fixes a regression where we lost the fun little animation on the 'hamburger' icon when viewing the site on mobile devices.

### CODE REVIEW

**Comment by mikeroySoft** [comment]
You can check out the workflow for editing docs directly on GitHub with a file from my fork here:
[https://mikeroysoft.github.io/photon/docs/overview/whats-new/](https://mikeroysoft.github.io/photon/docs/overview/whats-new/) 

Just click 'edit on GitHub' in the upper right corner. (this link is not present on the live Photon site currently)
The site is config'd to point to vmware/photon photon-hugo, so the experience would be identical, in case you wanted to see it.

File: DOCS-CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  Cleaned          |   0  
+
+Built in 1183 ms
+Watching for changes in /<local folder path>/photon/{assets,content,data,layouts,package.json,static,themes}
+Watching for config changes in /<local folder path>/photon/config.toml, /<local folder path>/photon/themes/photon-theme/config.toml
+Environment: "development"
+Serving pages from memory
+Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
+Web Server is available at //localhost:1313/ (bind address 127.0.0.1)
+Press Ctrl+C to stop
```
Comment (ssahani) [inline DOCS-CONTRIBUTING.md:155]: http missing? ```//localhost:1313 -> http://localhost:1313```

bind address 127.0.0.1 -> binds to loopback address 127.0.0.1 and port 1313

File: DOCS-CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Once those are installed, you should be able to run `hugo` to build the site locally or `hugo server` to run the site in a live-reload local environment.
+
+#### Building and Reviewing your Changes Locally
+
+You can simply make changes to the Markdown content without needing to build the entire site locally. 
+
+However, if you want to review the changes locally in what it's final form should look like, you would run the `hugo server` command from the root of the repo.
+
+`hugo server` will build the site in real-time and present it to you at //localhost:1313.  Any changes that happen to markdown or design content will automatically be rebuild and loaded into the site when it detects that files have changed.
+
```
Comment (ssahani) [inline DOCS-CONTRIBUTING.md:132]: http missing? //localhost:1313 -> http://localhost:1313. 
Extra space after period ```.  Any changes that happen```

**Review by ssahani — COMMENTED** [review]
Looks great to me. Just minor nits

**Comment by dcasota** [comment]
Awesome * master piece ! multi-language, "super simple way" editor, architecture support,..a must read.

**Comment by dcasota** [comment]
Is "super simple way edit" backward compatibility of Photon 1.0-2.0 docs supported with hugo? check this https://mikeroysoft.github.io/photon/assets/files/html/1.0-2.0/Running-Photon-OS-on-Google-Compute-Engine.html

File: DOCS-CONTRIBUTING.md (hunk 1 above)
Comment (mikeroySoft) [reply DOCS-CONTRIBUTING.md:155]: This is just the output from the `hugo` command verbatim

File: DOCS-CONTRIBUTING.md (hunk 2 above)
Comment (mikeroySoft) [reply DOCS-CONTRIBUTING.md:132]: yah it'd be fine to use http:, I was just referencing what `hugo` output looks like (which says //localhost:1313)

**Comment by mikeroySoft** [comment]
> Is "super simple way edit" backward compatibility of Photon 1.0-2.0 docs supported with hugo? check this https://mikeroysoft.github.io/photon/assets/files/html/1.0-2.0/Running-Photon-OS-on-Google-Compute-Engine.html

Negative... the 'Old' 1.0, 2.0 and 3.0 docs are not editable in the same way.

I mean, they _might_ be, but we deprecated that whole process and I am internally uninterested in reviving it.
So, they're just there for archive purposes and to make sure Google links don't 404.

The old site was generated using a process which no longer works, so they're frozen in time (hence the need to get on to a new publishing platform, which is what the new docs are)
I tried mucking about with the source earlier on, but  it's kind of a mess with how Gitbook organized things... I couldn't even add a 'click to go to the latest version of the docs' button without having to make changes on each page individually.

For 3.0, updates should be made to "/content/end/docs-v3/".

**Comment by dcasota** [comment]
@mikeroySoft thank you for the clarification. How can volunteers help? Is manually maintaining docs 1.0-2.0 acceptable or contra-productive?

**Comment by mikeroySoft** [comment]
We shouldn't be touching the older v1, v2 docs at all really. They're just here for archival reasons.

**Pull request merged**